src/web/app/pages-admin/admin-search-page/admin-search-page.component.spec.ts

```diff
@@ -65,9 +128,9 @@ describe('AdminSearchPageComponent', () => {
     }));
 
     const spyStatusMessageService: any = spyOn(statusMessageService, 'showErrorToast').and.callFake(
-      (args: string): void => {
-        expect(args).toEqual('This is the error message');
-      });
+        (args: string): void => {
+          expect(args).toEqual('This is the error message');
+        });
```

Comment:
Similarly here

Reply:
@madanalogy Thanks for the suggestion, made the changes 👍